tests/PhpUnits/Helpers/ValidatorTest.php

```diff
@@ -215,7 +214,7 @@ public function testValidateUsernameCharactersThrowError_01(): void {
 
 		public function testValidateUsernameSuccess(): void {
 			$this->assertTrue(
-				Validator::ValidateUsername("rawadymario")
```

Comment:
Here set the username to be: "digital_splash" or "digitalsplash" (Depending if we allow underscores in the username or not)